test/CMakeLists.txt

```diff
@@ -298,6 +298,11 @@ else ()
           COMMAND logconcave_sampling_test -tc=uld)
   add_test(NAME logconcave_sampling_test_exponential_biomass_sampling
           COMMAND logconcave_sampling_test -tc=exponential_biomass_sampling)
+  add_test(NAME logconcave_sampling_test_nuts_hmc_truncated
```

Comment:
I think it would be helpful to also add a C++ example of NUTS.

Reply:
@vissarion @TolisChal  I have already opened this PR to Tolis' fork with a NUTS example: https://github.com/TolisChal/volume_approximation/pull/29

Reply:
Great! You also resolved the `mkl` linking issue in examples ;-)